jscomp/build_tests/utils.js

```diff
@@ -0,0 +1,51 @@
+const child_process = require("child_process");
+
+const signals = {
+  SIGINT: 2,
+  SIGQUIT: 3,
+  SIGKILL: 9,
+  SIGTERM: 15,
+};
+
+/**
+ * @param {string} command
+ * @param {Array<string>} args
+ * @param {child_process.SpawnOptions} [options]
+ */
+async function exec(command, args, options) {
+  const stdoutChunks = [];
+  const stderrChunks = [];
+
+  const subprocess = child_process.spawn(command, args, {
+    stdio: ["ignore", "pipe", "pipe"],
+    ...options,
+  });
+
+  subprocess.stdout.on("data", chunk => {
+    stdoutChunks.push(chunk);
+  });
+
+  subprocess.stderr.on("data", chunk => {
+    stderrChunks.push(chunk);
+  });
+
+  return await new Promise((resolve, reject) => {
+    subprocess.once("error", err => {
+      reject(err);
+    });
+
+    subprocess.once("close", (exitCode, signal) => {
+      const stdout = Buffer.concat(stdoutChunks).toString("utf8");
+      const stderr = Buffer.concat(stderrChunks).toString("utf8");
+
+      let code = exitCode ?? 1;
+      if (signals[signal]) {
+        code = signals[signal] + 128;
```

Comment:
Can you add a comment here where the `+ 128` is coming from?

Reply:
Adding 128 to signal number is an old POSIX practice, which also Node.js follows: https://nodejs.org/api/process.html#exit-codes

IDK where it is exactly coming from, but there aren't many hints other than that it's an "old practice." I think the comment would be a bit verbose.